index/start-or-resume.njk

```diff
@@ -37,7 +37,11 @@
                 },
                 {
                     "value": "resume",
-                    "text": "Continue a saved application"
+                    "text": "Continue a saved vvpplication",
```

Comment:
Should be 'application' not 'vvpplication'

Reply:
Changed